infrastructure/terraform/main.tf

```diff
@@ -0,0 +1,68 @@
+resource "azurerm_resource_group" "main" {
+  name     = local.resource_group_name
+  location = local.region
+}
+
+module "app-key-vault" {
+  source = "../modules/dtos-devops-templates/infrastructure/modules/key-vault"
+
+  name                                             = "kv-${var.app_short_name}-${var.environment}"
+  resource_group_name                              = azurerm_resource_group.main.name
+  enable_rbac_authorization                        = true # TODO: make true by default?
```

Comment:
Noticed there's some TODOs left here... I would definitely be in favour of defaulting params that we devs don't need to worry about. Then we can focus on the settings that are specific to our app.

Reply:
Noted, we'll improve the modules